app/models/education_benefits_claim.rb

```diff
@@ -37,6 +37,10 @@ def parsed_form
     @parsed_form ||= JSON.parse(form)
   end
 
+  def confirmation_number
+    "vets_gov_#{self.class.to_s.underscore}_#{id}"
```

Comment:
Does this get displayed to the user? Is there any reason not to create a random uuid for this? My assumption is that this is used to communicate with the call center, and I would be concerned about giving a sequential identifier without knowing what other checks the call center does to verify identity